test/helpers/envtest.go

```diff
@@ -97,15 +120,64 @@ type TestEnvironment struct {
 // This function should be called only once for each package you're running tests within,
 // usually the environment is initialized in a suite_test.go file within a `BeforeSuite` ginkgo block.
 func NewTestEnvironment() *TestEnvironment {
+
+	// initialize webhook here to be able to test the envtest install via webhookOptions
+	// This should set LocalServingCertDir and LocalServingPort that are used below.
+	initializeWebhookInEnvironment()
+
 	if _, err := env.Start(); err != nil {
 		panic(err)
 	}
 
-	mgr, err := manager.New(env.Config, manager.Options{
+	env.Config.QPS = 1000
+	env.Config.Burst = 10000
```

Comment:
A comment on why we've chosen to set these would probably be useful to have

Reply:
Added by @randomvariable for testing, but I am not sure about the reason

Reply:
Yeah we shouldn't need these, they're already set pretty high in envtest defaults

Reply:
The problem is that by removing them, test was failing ...